draft-ietf-masque-connect-udp-listen.md

```diff
@@ -124,12 +124,15 @@ defined in {{compression}}.
 
 This extension leverages context IDs (see {{Section 4 of CONNECT-UDP}}) to
 compress the target IP address and port when encoding datagrams on the wire.
-Endpoint start by registering a context ID and the IP/ports it's associated
+Endpoint starts by registering a context ID and the IP/ports it's associated
```

Comment:
That sounds a little weird grammar-wise? Maybe `Each endpoint starts` or perhaps even better `Endpoints start`